docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -434,6 +444,85 @@ Stow Result
 This parameter is available on modules that support :ref:`manual_stow_position`. It reports on how the previous stow attempt ended. See the
 :ref:`stow_result` section for more information.
 
+Module ID
```

Comment:
Similar thing as before, maybe worth putting a note or something on these guys saying that they are new and not on all the releases yet, coming soon kind of thing.